EnglishWebRoot/school/pubbly_engine/latest/js/app/_AppModifications.js

```diff
@@ -1,4 +1,41 @@
 // Main class for all android/ios app modifications (assuming cordova)
+var endTime, startTime;
+function Calculate_time() {
+
+             var url = window.location.pathname.split("/");
+             url.pop(); // index.html
+             var book = url.pop();
+             var subject = url.pop();
+             var Module_name = url.pop();
+
+             function removeSpecialChars(str) {
+               return str.replace(/%20/g, "    ");
+             }
+             var Module_Name= removeSpecialChars(Module_name);
+             var Level_Name= removeSpecialChars(subject);
+             var Lesson_Name= removeSpecialChars(book);
+
+             endTime = new Date();
+             var timeDiff = endTime - startTime; //in seconds
+             timeDiff /= 1000;
+             var Elapsed_Time = Math.round(timeDiff);
+             console.log(Elapsed_Time + " Elapsed_Time");
+
+             document.addEventListener("deviceready", function(){
+             window.FirebasePlugin.setAnalyticsCollectionEnabled(true);
+             window.FirebasePlugin.setScreenName("Module_Play_Status");
+
+            if(Lesson_Name=="Epic    Quest")

```

Comment:
Use if(Lesson_Name.toLowerCase() === "epic quest")

Reply:
Added this code.